src/sagas/wallet.js

```diff
@@ -172,13 +173,14 @@ export function* startWallet(action) {
 
     if (!(words || xpub)) {
       const accessData = yield storage.getAccessData();
-      xpriv = cryptoUtils.decryptData(accessData.mainKey, pin);
+      xpriv = cryptoUtils.decryptData(accessData.acctPathKey, pin);
+      authxpriv = cryptoUtils.decryptData(accessData.authKey, pin);
```

Comment:
note: To start a wallet-service you can use the seed or the account path xpriv along with the auth path key.